e3db/client.py

```diff
@@ -763,13 +766,15 @@ def backup(self, client_id, registration_token):
         # credentials must be json encoded in order to decode
         # properly in the Tozny Dashboard.
         credentials = {
-            'version': '1',
+            'version': '2',
```

Comment:
I think we should have a logic switch here to verify signing keys or else this will get set to version 2 as default

Reply:
think through how this will be used. It only gets called at registration time, and by the time this feature branch(es) are merged in any client created will be a v2 client

Reply:
Something like 

```
if public_signing_key = nil:
    version = 1
else: 
    version = 2
```

Is something I original had in the commit before making the pr, but paused on it in case there was a preference to default to 2 and instead determine authentication on a per-method basis as per Levi's comment in standup?

Reply:
If this is being called for registration, we should only be creating v2 clients. I think this is fine to default to 2

Reply:
@Mrhea thats (each method will determine its auth method)) an orthogonal concern for this discussion on which version to use when backing up a newly registered client